packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -467,7 +425,7 @@ export function mapStateToProps(state: ReduxState): TReduxProps {
   };
 }
 
-export function mapDispatchToProps(dispatch: Dispatch<ReduxState>): TDispatchProps {
+export function mapDispatchToProps(dispatch: Dispatch<ReduxState>) {
```

Comment:
With functional components, you can use hooks like `useDispatch` instead of `mapDispatchToProps`. Similarly, `useSelector` for `mapStateToProps`. That would be a better approach, imo